sites/telecablesat.fr/telecablesat.fr_fr.channels.xml

```diff
@@ -0,0 +1,288 @@
+    <channel lang="fr" xmltv_id="" site_id="389">Canal C</channel>
+    <channel lang="fr" xmltv_id="" site_id="32">Canal J</channel>
+    <channel lang="fr" xmltv_id="" site_id="936">Canal Z</channel>
+    <channel lang="fr" xmltv_id="" site_id="386">Canal Zoom</channel>
+    <channel lang="fr" xmltv_id="" site_id="34">Canal+</channel>
+    <channel lang="fr" xmltv_id="" site_id="376">Canal+ Antilles</channel>
+    <channel lang="fr" xmltv_id="" site_id="33">Canal+ Cinéma</channel>
+    <channel lang="fr" xmltv_id="" site_id="30">Canal+ Décalé</channel>
+    <channel lang="fr" xmltv_id="" site_id="657">Canal+ Family</channel>
+    <channel lang="fr" xmltv_id="" site_id="1563">Canal+ Séries</channel>
+    <channel lang="fr" xmltv_id="" site_id="35">Canal+ Sport</channel>
+    <channel lang="fr" xmltv_id="" site_id="24">Canvas</channel>
+    <channel lang="fr" xmltv_id="" site_id="36">Cartoon Network</channel>
+    <channel lang="fr" xmltv_id="" site_id="318">CGTN</channel>
+    <channel lang="fr" xmltv_id="" site_id="38">Chasse et pêche</channel>
+    <channel lang="fr" xmltv_id="" site_id="1399">Chérie 25</channel>
+    <channel lang="fr" xmltv_id="" site_id="287">Ciné+ Classic</channel>
+    <channel lang="fr" xmltv_id="" site_id="437">Ciné+ Classic Belgique</channel>
+    <channel lang="fr" xmltv_id="" site_id="285">Ciné+ Club</channel>
+    <channel lang="fr" xmltv_id="" site_id="283">Ciné+ Emotion</channel>
+    <channel lang="fr" xmltv_id="" site_id="401">Ciné+ Famiz</channel>
+    <channel lang="fr" xmltv_id="" site_id="284">Ciné+ Frisson</channel>
+    <channel lang="fr" xmltv_id="" site_id="317">Ciné+ Frisson Belgique</channel>
+    <channel lang="fr" xmltv_id="" site_id="282">Ciné+ Premier</channel>
+    <channel lang="fr" xmltv_id="" site_id="294">Ciné+ Premier Belgique</channel>
+    <channel lang="fr" xmltv_id="" site_id="2781">Clique TV</channel>
+    <channel lang="fr" xmltv_id="" site_id="50">Club RTL</channel>
+    <channel lang="fr" xmltv_id="" site_id="226">CNEWS</channel>
+    <channel lang="fr" xmltv_id="" site_id="53">CNN</channel>
+    <channel lang="fr" xmltv_id="" site_id="54">Comédie+</channel>
+    <channel lang="fr" xmltv_id="" site_id="2752">Comedy Central</channel>
+    <channel lang="fr" xmltv_id="" site_id="2037">Crime District</channel>
+    <channel lang="fr" xmltv_id="" site_id="458">CSTAR</channel>
+    <channel lang="fr" xmltv_id="" site_id="13">Das Erste</channel>
+    <channel lang="fr" xmltv_id="" site_id="57">Demain TV</channel>
+    <channel lang="fr" xmltv_id="" site_id="400">Discovery Channel</channel>
+    <channel lang="fr" xmltv_id="" site_id="2324">Discovery Family</channel>
+    <channel lang="fr" xmltv_id="" site_id="2184">Discovery Investigation</channel>
+    <channel lang="fr" xmltv_id="" site_id="1374">Discovery Science</channel>
+    <channel lang="fr" xmltv_id="" site_id="58">Disney Channel</channel>
+    <channel lang="fr" xmltv_id="" site_id="1967">Disney Channel Wallonia</channel>

```

Comment:
```suggestion
    <channel lang="fr;nl" xmltv_id="" site_id="1967">Disney Channel Wallonia</channel>
```
Both Flemish Dutch and French

Reply:
This attribute specifies only the language of the program, not the broadcast, and in this case it is French:

<img width="857" alt="image" src="https://user-images.githubusercontent.com/7253922/157731434-66227199-ab90-4d7e-923b-4c428b7379a9.png">

Reply:
https://en.wikipedia.org/wiki/Disney_Channel_(Belgian_TV_channel)

Reply:
I meant the guide.